Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -2577,7 +2583,7 @@ describe(
       return testBackFaceCulling(withoutBatchTableUrl, setViewOptions);
     });
 
-    it("renders glTF tileset when back face culling is disabled", function () {
+    xit("renders glTF tileset when back face culling is disabled", function () {
```

Comment:
Similarly here, though the link is `https://github.com/CesiumGS/cesium/issues/10056`